### PR TITLE
Fixes #33773 - use correct fields for translations

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Comment by upadhyeammit** [comment]
@tbrisker I have not updated the actual translations yet, after confirming changes are correct will do the update. Request to review, thank you!

File: locale/de/foreman.po [hunk 1]
```diff
@@ -3238,7 +3238,7 @@ msgid "Failed to get a new realm OTP. Terminating the build!"
 msgstr "Fehler beim Abrufen eines neuen Bereichs-OTP. Build wird abgebrochen!"
 
 msgid "Failed to import %{klass} for %{name}: doesn't exists in our database - ignoring"
-msgstr "Fehler beim Importieren der Klasse %{class} for %{name}: existiert nicht in der Datenbank - wird ignoriert"
+msgstr "Fehler beim Importieren der Klasse %{kclass} for %{name}: existiert nicht in der Datenbank - wird ignoriert"
```
Comment (tbrisker) [inline locale/de/foreman.po:3241]:
```suggestion
msgstr "Fehler beim Importieren der Klasse %{klass} for %{name}: existiert nicht in der Datenbank - wird ignoriert"
```

File: locale/fr/foreman.po [hunk 2]
```diff
@@ -251,13 +251,13 @@ msgstr[0] "%{name} a %{num_tag} classe"
...
 msgid "%{object_name} is a %{object_class}, expected a subnet"
-msgstr "%{object_name} est un %{host}, on s’attendait à un sous-réseau"
+msgstr "%{object_name} est un %{object_class}, on s’attendait à un sous-réseau"
 
 msgid "%{os} medium was not set for host '%{host}'"
 msgstr "Medium %%s non défini pour l'hôte '%{host}'"
 
 msgid "%{record} is being used by a hidden %{what} resource"
-msgstr "%<b> est utilisé par la ressource %{version} cachée"
+msgstr "%<b> est utilisé par la ressource %{what} cachée"
```
Comment (tbrisker) [inline locale/fr/foreman.po:260]:
```suggestion
msgstr "%{record} est utilisé par la ressource %{what} cachée"
```

File: locale/fr/foreman.po [hunk 3]
```diff
@@ -251,13 +251,13 @@ msgstr[0] "%{name} a %{num_tag} classe"
 msgstr[1] "%{name} a %{num_tag} classes"
 
 msgid "%{object_name} is a %{object_class}, expected a subnet"
-msgstr "%{object_name} est un %{host}, on s’attendait à un sous-réseau"
+msgstr "%{object_name} est un %{object_class}, on s’attendait à un sous-réseau"
 
 msgid "%{os} medium was not set for host '%{host}'"
 msgstr "Medium %%s non défini pour l'hôte '%{host}'"
```
Comment (tbrisker) [inline locale/fr/foreman.po:257]:
```suggestion
msgstr "Medium %{os} non défini pour l'hôte '%{host}'"
```

File: locale/fr/foreman.po [hunk 4]
```diff
@@ -251,13 +251,13 @@ msgstr[0] "%{name} a %{num_tag} classe"
...
 
 msgid "%{record} is being used by a hidden %{what} resource"
-msgstr "%<b> est utilisé par la ressource %{version} cachée"
+msgstr "%<b> est utilisé par la ressource %{what} cachée"
 
 msgid "%{record} is used by %{what}"
 msgstr "%s utilisé"
```
Comment (tbrisker) [inline locale/fr/foreman.po:263]: this one also seems wrong. seems there are a lot of %s, %%s etc scattered around

File: locale/fr/foreman.po [hunk 5]
```diff
@@ -2947,7 +2947,7 @@ msgid "Error generating IP: %s"
 msgstr "Erreur de génération de l'adresse IP : %s"
 
 msgid "Error has occurred while communicating with %{cr}: %{e}"
-msgstr "Une erreur est survenue lors de la communication avec %%s : %{host}"
+msgstr "Une erreur est survenue lors de la communication avec %%s : %{e}"
```
Comment (tbrisker) [inline locale/fr/foreman.po:2950]:
```suggestion
msgstr "Une erreur est survenue lors de la communication avec %{cr} : %{e}"
```

File: locale/fr/foreman.po [hunk 6]
```diff
@@ -3190,13 +3190,13 @@ msgid "Failed to acquire IP addresses from compute resource for %s"
...
 msgid "Failed to cancel pending build for %{hostname} with the following errors: %{errors}"
-msgstr "Échec de l'annulation de la construction en cours pour %{host} avec les erreurs suivantes : %{errors}"
+msgstr "Échec de l'annulation de la construction en cours pour %{hostname} avec les erreurs suivantes : %{errors}"
 
 msgid "Failed to configure %{host} to boot from %{device}: %{e}"
 msgstr "Échec de la configuration de %{host} pour démarrer à partir de %{device}: : %{e}"
 
 msgid "Failed to create %{name}'s realm entry: %{e}"
-msgstr "Impossible de créer l'entrée du domaine %%s : %{host}"
+msgstr "Impossible de créer l'entrée du domaine %%s : %{e}"
```
Comment (tbrisker) [inline locale/fr/foreman.po:3199]:
```suggestion
msgstr "Impossible de créer l'entrée du domaine %{name} : %{e}"
```

File: locale/fr/foreman.po [hunk 7]
```diff
@@ -4888,22 +4888,22 @@ msgid "Interval or subscription option is missing"
...
 msgid "Invalid %{assoc} selection, you must select at least one of yours and have '%{perm}' permission."
-msgstr "Sélection %%s non valide. Vous devez sélectionner au moins une des vôtres et disposer de l'autorisation '%{name}'."
+msgstr "Sélection %%s non valide. Vous devez sélectionner au moins une des vôtres et disposer de l'autorisation '%{perm}'."
 
 msgid "Invalid Google JSON key. Please verify client email & private key options are present"
 msgstr "Clé JSON de Google invalide. Veuillez vérifier que les options email et de clé privée du client soient présentes"
 
 msgid "Invalid architecture '%{arch}' for '%{os}'"
-msgstr "Architecture invalide '%%s' pour '%{host}'"
+msgstr "Architecture invalide '%%s' pour '%{os}'"
```
Comment (tbrisker) [inline locale/fr/foreman.po:4897]:
```suggestion
msgstr "Architecture invalide '%{arch}' pour '%{os}'"
```

**Review by tbrisker — CHANGES_REQUESTED** [review]
looks like the french version needs quite a lot more work, there's lots of mismatching keys there

**Comment by tbrisker** [comment]
@upadhyeammit we probably need to correct the original translation in transifex and/or memsource so these fixes aren't overriden on the next pull

**Comment by upadhyeammit** [comment]
The RH translation team is working on the fixes, hence I am closing the pr. Shall raise new pr once we have correct translations on transifex.

**Pull request closed without merging**